sdk/src/main/proto/trace/v1/trace_config.proto

```diff
@@ -37,26 +37,20 @@ message TraceConfig {
   int64 max_number_of_attributes = 4;
 
   // The global default max number of annotation events per span.
-  int64 max_number_of_annotations = 5;
+  int64 max_number_of_timed_events= 5;
 
-  // The global default max number of message events per span.
-  int64 max_number_of_message_events = 6;
+  // The global default max number of attributes per timed event.
+  int64 max_number_of_attributes_per_timed_event = 6;
```

Comment:
do we need separate one for events and for annotations? They have a common base class, but not necessarily the semantics. I believe we may want more attributes on events and less on annotations.

Reply:
we no-longer have an "Annotation". Everything is an event. Do you want to make this separation?

Reply:
I was confused. Sorry.